test/test_adcs_handler.c

```diff
@@ -123,15 +138,113 @@ void test_ADCS_common_commands(void)
     command[1] = msg_l;
     command[2] = ign_hole;
     send_uart_telecommand_ExpectAndReturn(command, 3, ADCS_OK);
-    TEST_ASSERT_EQUAL_INT(ADCS_OK, ADCS_initiate_download_burst(msg_l, ign_hole));
+    TEST_ASSERT_EQUAL_INT(ADCS_OK, ADCS_initiate_download_burst(msg_l,
+ign_hole));
     //ADCS_invalid_TC_ID
-    
+
     free(command);
     command = NULL;
+}*/
+
+void test_ADCS_common_telemetry(void) {
+  TEST_IGNORE_MESSAGE("Need to Implement ADCS_CommonTelemetry");
 }
 
-void test_ADCS_common_telemetry(void)
-{
-    TEST_IGNORE_MESSAGE("Need to Implement ADCS_CommonTelemetry");
+void test_ADCS_BootLoader_telecommands(void) {
```

Comment:
I hope it's obvious what's happening here. I'll make a note here for myself to add some comments. But let me know if there's a question.